packages/wrangler/src/config/config.ts

```diff
@@ -30,6 +30,8 @@ export type RawConfig = Partial<ConfigFields<RawDevConfig>> &
 export interface ConfigFields<Dev extends RawDevConfig> {
 	configPath: string | undefined;
 
+	script_metadata_binding: string | undefined;
```

Comment:
Let's rename this to `metadata_binding`, latest PR's in the backend seem to have changed the name to avoid redundancy (worker vs. script vs. service, etc).

Reply:
This file is the one I mentioned for the payload:
https://github.com/cloudflare/wrangler2/blob/main/packages/wrangler/src/create-worker-upload-form.ts

Reply:
> This file is the one I mentioned for the payload: https://github.com/cloudflare/wrangler2/blob/main/packages/wrangler/src/create-worker-upload-form.ts

Correct that is the one I updated for this PR to include the metadata_binding field: https://github.com/cloudflare/wrangler2/blob/2a81caeeb785d0aa6ee242297c87ba62dfba48e7/packages/wrangler/src/create-worker-upload-form.ts#L260-L271

Reply:
> Let's rename this to `metadata_binding`, latest PR's in the backend seem to have changed the name to avoid redundancy (worker vs. script vs. service, etc).

Will rename the binding to `metadata_binding` on what is sent to the backend. I think it should still be called `script_metadata_binding` or `worker_metadata_binding` to be explicit what the binding is for